musicbot/bot.py

```diff
@@ -5292,10 +5292,16 @@ async def cmd_clear(
         return Response(_D("Cleared all songs from the queue.", ssd_))
 
     @command_helper(
-        usage=["{cmd} [POSITION]"],
+        usage=[
+            "{cmd} <POSITION> <TO>\n"
+            + _Dd("    Remove song at POSITION.\n")
```

Comment:
Each form of usage should be its own list entry here. 
```python
usage=[
    "{cmd} [POSITION]\n"
    + _Dd("    Remove a song at the end of the queue or at [POSITION].\n"),

    "{cmd} <FROM> <TO>\n"
    + _Dd("    Remove songs from position FROM to position TO.\n"),
],
```
The square brackets denote an optional argument, as remove previously allowed.